content/userspace/apps/reference/dist/README.md

```diff
@@ -1,88 +1,66 @@
+For the moment, every live desk must support the running kernel version. Kernel backwards compatibility is an intended feature in the future, but for now, app developers will need to publish an app update supporting new kernels each time they're released.
 
-If you're publishing an app that expects another app to be installed in order to function, the best practice is to check in `+on-init` for the presence of that other app's desk. If it's not installed, your app should display a message to the user and a link to the app that they should install in order to support your app. App-install links are well-supported in Tlon's Landscape, a suite of user-facing applications developed by Tlon.
+Each desk defines its own filetypes (called `mark`s), in its `/mar` folder. There are no shared system marks that all userspace code knows, nor common libraries in `/lib` or `/sur` — each desk is completely self-contained.
 
-For the moment, every live desk must have the same Kelvin version as the kernel. Future kernels that know how to maintain backward compatibility with older kernels will also allow older desks, but no commitment has yet been made to maintain backward compatibility across kernel versions, so for the time being, app developers should expect to update their apps accordingly.
+It's common for a apps to interact with apps on other desks, and therefore need marks and libraries by the other desk's developer. Since direct cross-desk dependencies are not currently supported, the typical approach is for app developers to publish a "dev" folder on their git repo with the files need for developers interfacing with their app.
 
-Each desk defines its own filetypes (called `mark`s), in its `/mar` folder. There are no longer shared system marks that all userspace code knows, nor common libraries in `/lib` or `/sur` — each desk is completely self-contained.
+Most desks will want to include the `%base-dev` desk in the [urbit/urbit](https://github.com/urbit/urbit) repo so they can easily interact with system apps in the `%base` desk.
 
-It's common for a desk to want to use files that were originally defined in another desk, so that it can interact with agents on that desk. The convention is that if you're publishing an app that you expect other devs to build client apps for (on other desks), you split out a "dev desk" containing just the external interface to your desk. Typically, both your app desk and clients' app desks will sync from this dev desk.
+landscape app (those with frontends launched from a tile in Landscape), should also include `desk-dev` in the [Landscape repo](https://github.com/tloncorp/landscape). This desk includes the `%docket-0` mark, which the app needs in order to include a `/desk/docket-0` file.
```

Comment:
Typo: sentence should start with "Landscape apps…".

Reply:
Could clarify what you mean here by saying "…should also include `desk-dev` from the Landscape repo…" rather than "in the Landscape repo".